### ISSUE TITLE
for an editable table, editing cells to the right of no-edit cells can't be done

### ISSUE THREAD

**Issue by Stephen-Hazel** [Mottie/tablesorter#789]
first off, GREAT code.

I've got a tablesorter with editable widget.
When I use no-edit class on some cells, cells withOUT no-edit (regular editable cells) can't be edited.
So let's say you have no-edit regular regular no-edit.  Everything is groovy.
But if you have no-edit regular regular no-edit regular...  That last regular cell can't be edited or tabbed to.  Tabbing goes directly down to the next row even though there's an editable cell later on on the same row.

**Comment by Mottie** [comment]
Hi @Stephen-Hazel!

I have a feeling that the `editable_columns` option may not be including the columns you want to have editable. I set up [this demo](http://jsfiddle.net/Mottie/abkNM/4951/) to make sure.

The [editable widget demo](http://mottie.github.io/tablesorter/docs/example-widget-editable.html) has that option set to only include the first 3 columns:

```js
editable_columns : [0,1,2]
```

In the demo I shared above, it includes all columns:

```js
editable_columns : '0-5'
```

And it works without any issue for me.
